onnxruntime/core/providers/openvino/ov_versions/data_ops.cc

```diff
@@ -0,0 +1,879 @@
+    {VERSION_2020_4,{"All"},"ReduceSum"},
+    {VERSION_2020_4,{"CPU", "MYRIAD"},"ReduceSumSquare"},
+    {VERSION_2020_4,{"All"},"Relu"},
+    {VERSION_2020_4,{"CPU"},"Resize"},
+    {VERSION_2020_4,{"All"},"Reshape"},
+    {VERSION_2021_1,{"All"},"RoiAlign"},
+    {VERSION_2021_2,{"MYRIAD"},"Round"},
+    {VERSION_2021_1,{"MYRIAD"},"Scatter"},
+    {VERSION_2021_2,{"MYRIAD"},"ScatterElements"},
+    {VERSION_2020_4,{"CPU", "GPU"},"Selu"},
+    {VERSION_2020_4,{"All"},"Shape"},
+    {VERSION_2020_4,{"All"},"Sigmoid"},
+    {VERSION_2020_4,{"CPU"},"Sign"},
+    {VERSION_2020_4,{"CPU"},"Sign"},
+    {VERSION_2020_4,{"CPU"},"Sinh"},
+    {VERSION_2020_4,{"MYRIAD"},"SinFloat"},
+    {VERSION_2020_4,{"All"},"Slice"},
+    {VERSION_2020_4,{"All"},"Softmax"},
+    {VERSION_2020_4,{"All"},"SpaceToDepth"},
+    {VERSION_2020_4,{"All"},"Split"},
+    {VERSION_2020_4,{"All"},"Sqrt"},
+    {VERSION_2020_4,{"All"},"Squeeze"},
+    {VERSION_2020_4,{"CPU"},"Softsign"},
+    {VERSION_2020_4,{"All"},"Sub"},
+    {VERSION_2020_4,{"All"},"Sum"},
+    {VERSION_2020_4,{"CPU", "GPU"},"Tan"},
+    {VERSION_2020_4,{"All"},"Tanh"},
+    {VERSION_2021_2,{"MYRIAD"},"Tile"},
+    {VERSION_2020_4,{"All"},"Transpose"},
+    {VERSION_2020_4,{"All"},"TopK"},
+    {VERSION_2020_4,{"All"},"Unsqueeze"},
+    {VERSION_2021_1,{"CPU", "GPU"},"Upsample"},
+    {VERSION_2021_2,{"MYRIAD"},"Where"},
+    
+};
+
+void Capability::populate_types_supported() {
+
+    std::set<int> supported_types_initializer = {
+      ONNX_NAMESPACE::TensorProto_DataType::TensorProto_DataType_BOOL,
+      ONNX_NAMESPACE::TensorProto_DataType::TensorProto_DataType_FLOAT,
+      ONNX_NAMESPACE::TensorProto_DataType::TensorProto_DataType_INT32,
+      ONNX_NAMESPACE::TensorProto_DataType::TensorProto_DataType_INT64,
+    };
+    
+    std::set<int> supported_types_vpu = {
+      ONNX_NAMESPACE::TensorProto_DataType::TensorProto_DataType_BOOL,
+      ONNX_NAMESPACE::TensorProto_DataType::TensorProto_DataType_FLOAT,
+      ONNX_NAMESPACE::TensorProto_DataType::TensorProto_DataType_INT32,
+      ONNX_NAMESPACE::TensorProto_DataType::TensorProto_DataType_INT16,
+      ONNX_NAMESPACE::TensorProto_DataType::TensorProto_DataType_INT8,
+      ONNX_NAMESPACE::TensorProto_DataType::TensorProto_DataType_UINT8,
+      ONNX_NAMESPACE::TensorProto_DataType::TensorProto_DataType_INT64,
+    };
+
+    std::set<int> supported_types_cpu = {
+      ONNX_NAMESPACE::TensorProto_DataType::TensorProto_DataType_BOOL,
+      ONNX_NAMESPACE::TensorProto_DataType::TensorProto_DataType_FLOAT,
+      ONNX_NAMESPACE::TensorProto_DataType::TensorProto_DataType_INT32,
+      ONNX_NAMESPACE::TensorProto_DataType::TensorProto_DataType_INT16,
+      ONNX_NAMESPACE::TensorProto_DataType::TensorProto_DataType_INT8,
+      ONNX_NAMESPACE::TensorProto_DataType::TensorProto_DataType_UINT8,
+      ONNX_NAMESPACE::TensorProto_DataType::TensorProto_DataType_INT64,
+    };
+
+    std::set<int> supported_types_gpu = {
```

Comment:
we need to add **ONNX_NAMESPACE::TensorProto_DataType::TensorProto_DataType_FLOAT16** for GPU and CPU

Reply:
resolved